AElf.Kernel/IAccountDataProvider.cs

```diff
@@ -6,7 +6,7 @@ namespace AElf.Kernel
     /// <summary>
     /// Data is stored associated with Account
     /// </summary>
-    public interface IAccountDataProvider
+    public interface IAccountDataProvider : ISerializable
```

Comment:
Data Provider is a Driver, should not be ISerializable, it's just a tool for data access